src/github.com/couchbase/sync_gateway/db/channel_cache.go

```diff
@@ -257,3 +307,140 @@ func (c *channelCache) prependChanges(changes LogEntries, changesValidFrom uint6
 		return 0
 	}
 }
+
+type lateLogEntry struct {
+	logEntry      *LogEntry
+	arrived       time.Time
+	listenerCount uint64
```

Comment:
Added comments for lateLogEntry attributes, including this one:
	listenerCount uint64       // Tracks the number of active changes feeds referencing this entry.  Used to identify when it's safe to prune